mastercraft.cmd

```diff
@@ -2037,6 +2100,7 @@ lack.material:
           }
      if "%discipline" = "remed" then
           {
+			   			   if "%order.type" = NULL then return

```

Comment:
Added this handling because sometimes you wuld get into order loop with NULL. This just cleans it up, stops a hang that happens sometimes.